src/resources/i18n/fr.json

```diff
@@ -130,7 +130,8 @@
     "searchbill": "Rechercher facture",
     "bill_chose": "Choisir une option",
     "managebills": "Manager Factures",
-    "billrecords": "Liste Factures"
+    "billrecords": "Liste Factures",
+    "visits": "Visites"
```

Comment:
Any updates in the internationalization resources that are not in the default language (**en**) should be done on [Transifex](https://www.transifex.com/informatici-senza-frontiere-onlus/openhospital-ui).